tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNetCore/UserEvents/UserEventsCommon.cs

```diff
@@ -49,7 +59,47 @@ internal static class UserEventsCommon
             return new string(stringChars, 0, 37);
         }
 
-        return null;
+        Log.Warning<int>("Couldn't anonymize user information (login or id), byteArray length was {BytesWritten}", bytesWritten);
```

Comment:
Yes, it should never happen but.. if it ever happened indeed it would be for every authenticated request (an incoming PR should collect user info for every authenticated request) so changed in 03da6074d4a68f6088e3c7119b6fb5c29b345c1d